apps/api/src/server/swagger.js

```diff
@@ -180,6 +180,26 @@ const document = {
 			filter1: 'Marketing',
 			filter2: 'Brochure'
 		},
+		documentsMetadataResponseBody: {
+			version: 1,
+			documentReference: 'a6f9f2e0-12c9-49b7-8a1c-3b5edc34dd99',
+			publishedDate: '',
+			caseRef: 'BC0210002',
+			documentName: '5',
+			blobStorageContainer: 'document-service-uploads',
+			blobStoragePath: '/application/BC010001/1111-2222-3333/my doc.pdf',
+			from: 'joe blogs',
+			receivedDate: 1_677_585_578,
+			size: 0,
+			fileType: '',
+			redacted: false,
+			status: 'awaiting_upload',
+			description: '',
+			documentReferenceNumber: '',
```

Comment:
should documentReferenceNumber be removed?  It looks like elsewhere you have replaced documentReferenceNumber with documentReference (which is also already in line 185 above).?